modules/core/src/main/lilaism/LilaLibraryExtensions.scala

```diff
@@ -125,3 +125,6 @@ trait LilaLibraryExtensions extends LilaTypes:
 
     // inline def unary_! = fua.map { !_ }(EC.parasitic)
     inline def not = fua.map { !_ }(EC.parasitic)
+
+  extension (condition: Boolean)
+    def thenList[A](value: => A): List[A] = if condition then List(value) else Nil
```

Comment:
already exists as the generic `so`.

```scala
false.so(List(42)) // Nil
```

Reply:
You don't think the reduction in parens is worth a new extension method?

Reply:
I think that's not generic enough to be made available throughout the entire code base. And I changed the way modules are listed (List[Option[Module]]) so it wouldn't be used as much anyway